Composer/packages/client/src/components/GetStarted/GetStartedNextSteps.tsx

```diff
@@ -0,0 +1,223 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React, { useEffect, useState } from 'react';
+import { useRecoilValue } from 'recoil';
+import formatMessage from 'format-message';
+
+import TelemetryClient from '../../telemetry/TelemetryClient';
+import { localBotsDataSelector } from '../../recoilModel/selectors/project';
+import { currentProjectIdState } from '../../recoilModel';
+import { ManageLuis } from '../ManageLuis/ManageLuis';
+import { ManageQNA } from '../ManageQNA/ManageQNA';
+import { dispatcherState, settingsState } from '../../recoilModel';
+import { mergePropertiesManagedByRootBot } from '../../recoilModel/dispatchers/utils/project';
+import { rootBotProjectIdSelector } from '../../recoilModel/selectors/project';
+import { navigateTo } from '../../utils/navigation';
+
+import { GetStartedTask } from './GetStartedTask';
+import { h3Style } from './styles';
+
+type GetStartedProps = {
+  requiresLUIS: boolean;
+  requiresQNA: boolean;
+};
+
+export type NextSteps = {
+  checked: boolean;
+  key: string;
+  description: string;
+  learnMore: string;
+  required?: boolean;
+  label: string;
+  onClick: (step?: NextSteps) => void;
+};
+
+export const GetStartedNextSteps: React.FC<GetStartedProps> = (props) => {
+  const projectId = useRecoilValue(currentProjectIdState);
+  const botProjects = useRecoilValue(localBotsDataSelector);
+  const botProject = botProjects.find((b) => b.projectId === projectId);
```

Comment:
Nit: You probably want this to be a useMemo with dependencies on botProjects and projectId.  With the useRecoil value your control will re-render when either change, so this is not a big deal.